### PR TITLE
Initial 9P commit

### PR DESCRIPTION
**Pull request by fuchicar**
- Created zfs-9p directory
- Added create operation.
- Added provisional main function.

### CODE REVIEW

**Comment by elbing** [comment]
Looks good. Let's wait for setting up pull approve and perhaps travis to ensure this builds at every commit :-).

**Comment by sevki** [comment]
both pull approve and travis are enabled, both are now required to pass to be able to merge stuff into master, just like harvey main :+1:

**Comment by sevki** [comment]
testing :+1:

File: src/zfs-9p/zfs_operations.c [hunk 1]
```diff
@@ -0,0 +1,181 @@
...
+			 */
+			error = EOVERFLOW;
+			return error;
+		}
+	}
+
+	/*
+	 * Check permissions.
+	 */
+	if (!(vfs->fuse_attribute & FUSE_VFS_HAS_DEFAULT_PERM)) {
```
Comment (rminnich) [inline src/zfs-9p/zfs_operations.c]: it was making sense until I got here. how is the fuse_attribute needed? I lost that part.

**Comment by elbing** [comment]
LGTM

**Comment by rminnich** [comment]
I don't see the travis step.

**Comment by elbing** [comment]
Ah, right. is "This branch has no conflicts". travis.yml missing.

**Comment by elbing** [comment]
back to pending :-)

**Comment by sevki** [comment]
this was submitted before there was a travis check on this repo, so this should be closed and resubmitted to be merged in to the master branch, with a .travis.yml file

**Comment by sevki** [comment]
@fuchicar I'm closing this, so you can open a new PR to merge in to `master` :) this is trying to merge in to `maint` branch

**Pull request closed without merging**